docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -361,28 +361,33 @@
             "properties": {
               "url": {
                 "type": "string",
-                "description": "For data source plugins. URL to fetch the authentication token."
+                "description": "URL to fetch the authentication token."
+              },
+              "scopes": {
+                "type": "string",
+                "description": ""
               },
               "params": {
                 "type": "object",
-                "description": "For data source plugins. Parameters for the token authentication request.",
+                "description": "Parameters for the token authentication request.",
                 "additionalProperties": false,
                 "properties": {
                   "grant_type": {
                     "type": "string",
-                    "description": "For data source plugins. OAuth grant type."
+                    "description": "OAuth grant type",
+                    "enum": ["client_credentials"]
```

Comment:
Based on the code all of these parameters is actually dynamic why I think grant_type should not have the enum part:
https://github.com/grafana/grafana/blob/85fed1cc0b29b126f23b461e0383c67108f607fe/pkg/api/pluginproxy/access_token_provider.go#L107-L116